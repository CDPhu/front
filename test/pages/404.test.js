import renderer from 'react-test-renderer';
import Custom404 from '../../pages/404';

const renderTree = tree => renderer.create(tree);
describe('<Custom404>', () => {
  it('should render component', () => {
    expect(renderTree(<Custom404 />).toJSON()).toMatchSnapshot();
  });
  
});