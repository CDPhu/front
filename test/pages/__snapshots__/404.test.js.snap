// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Custom404> should render component 1`] = `
<div
  className="bg-bvpk-900 flex items-center justify-center h-screen overflow-hidden "
>
  <div
    className="px-6 py-4 rounded"
  >
    <div
      className="mb-2 text-white text-xl"
    >
       
      <span
        className="font-bold"
      >
        404
      </span>
       | Page not found
    </div>
  </div>
</div>
`;
